packages/components/src/components/icon/mt5/ic-mt5-zero-spread-platform.svg

```diff
@@ -0,0 +1 @@
-.27-4.32l-.72 1.674-.792 1.836c-.252.612-.498 1.2-.738 1.764-.24.552-.444 1.026-.612 1.422H13.36c-.168-.396-.372-.87-.612-1.422-.24-.564-.492-1.152-.756-1.764-.252-.612-.51-1.224-.774-1.836l-.72-1.674a106.97 106.97 0 00-.27 4.32c-.06 1.428-.108 2.748-.144 3.96H7.347c.048-.984.102-2.034.162-3.15l.216-3.33c.084-1.116.174-2.184.27-3.204.108-1.032.222-1.962.342-2.79h2.682zM32.038 14.526v2.394h-3.762V27h-2.808V16.92h-3.762v-2.394h10.332zM37.982 23.328a1.77 1.77 0 00-.234-.918c-.156-.276-.426-.504-.81-.684-.384-.18-.9-.318-1.548-.414-.648-.096-1.464-.144-2.448-.144.132-1.116.24-2.25.324-3.402.084-1.152.15-2.232.198-3.24h6.75v2.25h-4.5c-.024.432-.054.852-.09 1.26a18.04 18.04 0 01-.09 1.044c1.752.12 3.042.534 3.87 1.242.84.696 1.26 1.668 1.26 2.916 0 .576-.102 1.11-.306 1.602-.204.492-.51.918-.918 1.278-.408.36-.924.642-1.548.846-.612.204-1.332.306-2.16.306-.324 0-.666-.024-1.026-.072a9.615 9.615 0 01-1.026-.162c-.324-.06-.618-.126-.882-.198a3.463 3.463 0 01-.612-.216l.486-2.214c.3.132.696.264 1.188.396.492.12 1.08.18 1.764.18.828 0 1.428-.162 1.8-.486.372-.336.558-.726.558-1.17zM20.724 39.113c-.192.198-.42.456-.684.774-.264.312-.537.654-.82 1.026-.281.372-.56.756-.836 1.152-.276.396-.522.774-.738 1.134h3.177v1.179h-4.905v-.837c.168-.306.372-.645.612-1.017a41.8 41.8 0 01.765-1.134 27.726 27.726 0 011.566-2.07h-2.8v-1.18h4.663v.973zM23.643 38.069c.936 0 1.653.168 2.15.504.499.33.748.846.748 1.548 0 .438-.102.795-.306 1.07-.198.27-.486.484-.864.64.126.156.258.336.396.54.138.198.273.408.405.63.138.216.27.444.396.684.126.234.243.465.35.693h-1.574c-.114-.204-.231-.411-.351-.621-.114-.21-.234-.414-.36-.612-.12-.198-.24-.384-.36-.558-.12-.18-.24-.342-.36-.486h-.693v2.277h-1.404V38.23c.306-.06.62-.102.945-.126a12.3 12.3 0 01.882-.036zm.08 1.197c-.101 0-.194.003-.278.009a35.48 35.48 0 00-.225.018v1.692h.396c.528 0 .906-.066 1.134-.198.228-.132.342-.357.342-.675 0-.306-.117-.522-.351-.648-.228-.132-.567-.198-1.017-.198zM29.52 43.307c.198 0 .36-.015.486-.045.132-.036.237-.081.315-.135a.48.48 0 00.162-.207.722.722 0 00.045-.261c0-.204-.096-.372-.288-.504-.192-.138-.522-.285-.99-.441a8.642 8.642 0 01-.612-.243 2.45 2.45 0 01-.55-.351 1.754 1.754 0 01-.395-.513 1.635 1.635 0 01-.153-.738c0-.288.054-.546.162-.774.108-.234.26-.432.459-.594.198-.162.438-.285.72-.37.282-.09.6-.134.954-.134.42 0 .783.045 1.089.135.306.09.558.189.756.297l-.405 1.107a3.496 3.496 0 00-.585-.234 2.374 2.374 0 00-.765-.108c-.336 0-.58.048-.73.144-.143.09-.215.23-.215.423 0 .114.027.21.08.288.055.078.13.15.226.216.102.06.216.117.342.17.132.049.276.1.432.154.324.12.606.24.846.36.24.114.438.249.594.405.162.156.282.339.36.549.078.21.117.465.117.765 0 .582-.204 1.035-.612 1.359-.408.318-1.023.477-1.845.477-.276 0-.525-.018-.747-.054a4.527 4.527 0 01-.594-.117 3.727 3.727 0 01-.441-.153 3.616 3.616 0 01-.306-.153l.396-1.116c.186.102.414.195.684.279.276.078.612.117 1.008.117z" fill="#fff"/></g><defs><clipPath id="prefix__clip0_29689_37799"><path fill="#fff" d="M0 0h48v48H0z"/></clipPath></defs></svg>
```

Comment:
Why do we have this icon 2 times in this folder?

Reply:
Removed one of them